library/core/src/future/future.rs

```diff
@@ -91,11 +91,10 @@ pub trait Future {
     /// (memory corruption, incorrect use of `unsafe` functions, or the like),
     /// regardless of the future's state.
     ///
-    /// [`Poll::Pending`]: ../task/enum.Poll.html#variant.Pending
-    /// [`Poll::Ready(val)`]: ../task/enum.Poll.html#variant.Ready
-    /// [`Context`]: ../task/struct.Context.html
-    /// [`Waker`]: ../task/struct.Waker.html
-    /// [`Waker::wake`]: ../task/struct.Waker.html#method.wake
+    /// [`Poll::Pending`]: Poll::Pending
+    /// [`Poll::Ready(val)`]: Poll::Ready
+    /// [`Waker`]: crate::task::Waker
+    /// [`Waker::wake`]: crate::task::Waker::wake
     #[cfg_attr(not(bootstrap), lang = "poll")]
```

Comment:
@pietroalbini This line is generating a merge conflict so I'm going to rebase this and have this line be the same as `master`. Is that correct? Also, is there a section in the docs you can point me to on why this is changed at release time?

Reply:
Some documentation on why `cfg(boostrap)` and `cfg(not(bootstrap))` are needed is [on the rustc-dev-guide](https://rustc-dev-guide.rust-lang.org/building/bootstrapping.html#complications-of-bootstrapping). That line should become:

```rust
#[lang = "poll"]
```

Reply:
So was this a just-stabilized feature then?